LessonSix/src/test/java/ru/geekbrains/CheckOneAndFourTest.java

```diff
@@ -0,0 +1,32 @@
+package ru.geekbrains;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.extension.ParameterContext;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.converter.ArgumentConverter;
+import org.junit.jupiter.params.converter.ConvertWith;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.stream.Stream;
+
+class CheckOneAndFourTest implements ArgumentConverter {
+
+    @CsvSource(value = {
+            "true:1,4,1,4",
+            "false:1,1,1,1",
+            "false:4,4,4,4",
+            "false:5"},
+            delimiter = ':')
+
+    @ParameterizedTest
+    void CheckOneAndFour(boolean result, @ConvertWith(CheckOneAndFourTest.class) int[] array) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
Yes, my mistake